lib/index.js

```diff
@@ -3,9 +3,9 @@
 const npmCheck = require('./in');
 const createState = require('./state/state');
 
-function init(userOptions) {
-    return createState(userOptions)
-        .then(currentState => npmCheck(currentState));
+async function init(userOptions) {
+    const currentState = await createState(userOptions);
+    return npmCheck(currentState);
```

Comment:
Nice! 🙌 

Maybe just:

```suggestion
    return npmCheck(await createState(userOptions));
```